PetrolEngine.cpp

```diff
@@ -10,11 +15,18 @@ PetrolEngine::PetrolEngine(int power, float capacity, int gears)
     std::cout << __FUNCTION__ << std::endl;
 }
 
+PetrolEngine::~PetrolEngine() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
 void PetrolEngine::changeGear(int gear)
 {
-    // TODO: Add checking if gear is between -1 and gears_
-    // -1 is for REAR
-    // 0 is for NEUTRAL
+    if (gear < -1) {
```

Comment:
I suggest you tu use constexpr for values like -1

Reply:
done